sway/commands/output.c

```diff
@@ -0,0 +1,251 @@
+#define _XOPEN_SOURCE 500
+#include <ctype.h>
+#include <libgen.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strings.h>
+#include <unistd.h>
+#include <wordexp.h>
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "list.h"
+#include "log.h"
+#include "stringop.h"
+
+static char *bg_options[] = {
+	"stretch",
+	"center",
+	"fill",
+	"fit",
+	"tile",
+};
+
+struct cmd_results *cmd_output(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "output", EXPECTED_AT_LEAST, 1))) {
+		return error;
+	}
+	const char *name = argv[0];
+
+	struct output_config *output = calloc(1, sizeof(struct output_config));
```

Comment:
Would be simpler to just have a create func here rather than exposing `output_config_defaults()`.